consensus/scp/src/slot.rs

```diff
@@ -2626,7 +2626,7 @@ mod ballot_protocol_tests {
         }
 
         // Force ballot timeout timer to fire.
-        slot.next_ballot_at = Some(Instant::now() - Duration::from_secs(1));
+        slot.next_ballot_at = Some(Instant::now().checked_sub(Duration::from_secs(1)).unwrap());
```

Comment:
For those like me who are surprised to see this change, see https://github.com/rust-lang/rust-clippy/issues/9371

Reply:
Nice, that was one of my gripes with _safe_ rust. debug would always panic on underflow while release would wrap around
> Performs subtraction that returns None instead of wrapping around on underflow.
unintentional underflow and overflow are sources of many bugs in industry, very nice to see this warning on by default